lib/cn/cndb_omf.h

```diff
@@ -133,7 +133,7 @@ struct cndb_info_omf {
     char                cninfo_meta[];
 } HSE_PACKED;
 
-OMF_SETGET(struct cndb_info_omf, cninfo_fanout_bits, 32);
+OMF_SETGET(struct cndb_info_omf, cninfo_fanout, 32);
```

Comment:
This should be a CNDB OMF version change.  Same struct, but different meaning.

Reply:
I guess I should just rev everything up a notch...